eng/native/functions.cmake

```diff
@@ -1,3 +1,14 @@
+include(CheckCSourceCompiles)
+
+# License: https://github.com/clibs/cmocka/blob/master/cmake/Modules/COPYING-CMAKE-SCRIPTS
+# https://github.com/clibs/cmocka/blob/master/cmake/Modules/CheckCCompilerFlagSSP.cmake
```

Comment:
I'm not sure if we can just link to the license here, afaik we'd need to embed the license text.

@smx-smx what if we added an `CLR_CMAKE_DISABLE_FSTACK_PROTECTOR_STRONG` variable that you could set in your custom toolchain file instead?

Reply:
That could also work, but i wonder if this discussion even makes sense for the individual (one) line of code that is the check in question (but i'm not a lawyer).
The idea or concept behind the check could be implemented by anyone.

Reply:
yeah it seems pretty trivial but I'm not a lawyer either :)